pkg/machine/e2e/machine_test.go

```diff
@@ -26,7 +26,7 @@ const (
 )
 
 var (
-	tmpDir         = "/var/tmp"
+	tmpDir         = os.TempDir()
```

Comment:
this defaults to /tmp so it changes the default location for the VM image, given the image is quite large maybe it is better to not store it in tmpfs

Reply:
that is done with intent for perf